lang/main.json

```diff
@@ -130,7 +130,7 @@
         "etherpad": "Open / Close shared document",
         "documentOpen": "Open shared document",
         "documentClose": "Close shared document",
-        "shareRoom": "Share room",
+        "shareRoom": "Invite someone",
```

Comment:
Why was this change needed for chat?

Reply:
This button didn't have a label in the past as it was icon-only. I just thought it's an inconsistency, comparing it to the accessibilityLabel.shareRoom value. On the other hand "Share room" doesn't seem to be a good label, but you tell me, you're a native speaker. I'm happy to change it back if you feel so.

Reply:
I'm barely able to communicate in any language other than grunts. I'm okay with the change.